parameter_builder/test/test_copyright.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2015 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
```

Comment:
is there a reason you chose the apache license over the 3-clause BSD license we use in moveit2?

Reply:
There's no reason, I think I copy-pasted it from somewhere else

Reply:
Do you mind using the ament_copyright to put the bsd 3 clause license in all these files and get rid of the apache license for consistency sake?